tests/testmarker/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+
+go_library(
+    name = "testmarker_lib",
+    srcs = ["walker.go"],
+    embedsrcs = [".testmarker.yaml"],
+    importpath = "github.com/pingcap/tidb/tests/testmarker",
+    visibility = ["//visibility:private"],
+    deps = [
+        "@com_github_ghemawat_stream//:stream",
+        "@com_github_zabawaba99_go_gitignore//:go-gitignore",
+        "@in_gopkg_yaml_v2//:yaml_v2",
```

Comment:
why not v3?

Reply:
we use v2 on tidb and only parser uses v3